packages/imt.sol/package.json

```diff
@@ -22,8 +21,7 @@
         "@typechain/hardhat": "^6.1.2",
         "@types/chai": "^4.2.0",
         "@types/mocha": ">=9.1.0",
-        "@zk-kit/incremental-merkle-tree": "^0.4.1",
-        "circomlibjs": "^0.0.8",
+        "@zk-kit/imt": "workspace:packages/imt",
```

Comment:
Can you verify that this import is correct?

Reply:
Yes it's correct!